cpg-core/src/testFixtures/java/de/fraunhofer/aisec/cpg/TestUtils.kt

```diff
@@ -117,9 +117,17 @@ object TestUtils {
         files: List<File>?,
         topLevel: Path,
         usePasses: Boolean,
-        configModifier: Consumer<TranslationConfiguration.Builder>? = null
+        configModifier: Consumer<TranslationConfiguration.Builder>? = null,
+        jsonCompilationDatabase: File? = null
```

Comment:
I would avoid adding a separate parameter for the jsonCompilationDatabase here, because we will end up with one function for each config options. You should use the `configModifier` for that instead. That allows you to access the config builder in a lambda expression. 

`analyze(files, topLevel, usePasses) { it.useCompilationDatabase(this, that) }`

Reply:
Thanks for the hint!
I fixed it.